Document-pl/0x01-Foreword.md

```diff
@@ -1,26 +1,30 @@
+nie są bezcelowe, ale należy mieć na uwadzę żę mogą zostać ominięte przez sprawnego atakującego, dlatego nigdy nie powinny być wykorzystywane jako zamiennik dla kontroli bezpieczeństwa po stronie serwerowej. 
 
-Ogólnym celem MASVS jest zaoferowanie podstaw dla bezpieczeństwa mobilnych aplikacji (MASVS - L1), zarazem zezwalając na włączanie środków obrony wielowarstwowej (eng: defense in depth) (MASVS - L2) i ochrony przeciwko zagrożeniom po stronie klienckiej (MASVS-R). MASVS ma na celu zapewnienie poniższych:
+Ogólnym celem MASVS jest wskazanie podstaw dla bezpieczeństwa mobilnych aplikacji (MASVS - L1), jednocześnie zezwalając na włączanie zaawansowanych mechanizmów bezpieczeństwa (eng: defense in depth) (MASVS - L2) i ochrony przeciwko zagrożeniom po stronie klienckiej (MASVS-R). MASVS ma na celu dostarczenie poniższych:
 
-- Zapewnienie wymagań dla architektów oprogramowania i developerów chcących tworzyć bezpieczne mobilne aplikacje;
-- Zaoferować standard branży którym można testować raporty bezpieczeństwa mobilnych aplikacji
-- Przejrzystość roli mechanizmów ochrony w bezpieczeństwie mobilnym i zapewnienie wymagań do zweryfikowania ich efektywności;
-- Zapewnienie specyficznych zaleceń wobec tego jaki poziom bezpieczeństwa jest zalecany dla różnych przypadków użycia.
+- Dostarczenie wymagań dla architektów oprogramowania i developerów chcących tworzyć bezpieczne aplikacje mobilne;
+- Zaoferowanie standardu dla branży, który może zotać wykorzystany jako podstawa do utworzenia raportu z testów bezpieczeństwa aplikacji mobilnych;
+- Wyjaśnienie znaczenia mechanizmów ochronnych w kontekście bezpieczeństwie aplikacji mobilnych i zapewnienie wymagań do zweryfikowania ich efektywności;
+- Dostarczenie konretnych rekomendacji wobec tego jaki poziom bezpieczeństwa jest zalecany dla różnych aplikacji.
 
-Jesteśmy świadomi tego, że 100% konsensus w branży jest niemożliwy do osiągnięcia. Mimo to, mamy nadzieję, że MASVS będzie użyteczny w dostarczeniu porad we wszystkich fazach tworzenia aplikacji mobilnych i ich testowania. MASVS będzie rozwijać się z biegiem czasu i przyjmiemy jakikolwiek wkład bądź sugestie.
+Jesteśmy świadomi tego, że 100% konsensus w branży nie jest możliwy do osiągnięcia. Mimo to, mamy nadzieję, że MASVS będzie użyteczny w dostarczeniu wskazówk we wszystkich fazach tworzenia aplikacji mobilnych i ich testowania. MASVS będzie rozwijać się z biegiem czasu i przyjmiemy jakikolwiek wkład bądź sugestie.
 
-Stworzone przez Bernharda Muellera
+Słowa wstępu zostały napisane przez Bernharda Muellera.
+
+Tłumaczenie na język polski zostało wykonane przez: TO DO 
```

Comment:
Nie widziałem w żadnych tłumaczeniach tej linijki niestety :/ Jedynie w Frontispiece.md jest miejsce na to